build.gradle

```diff
@@ -289,6 +328,15 @@ allprojects {
 //
 // Command line args can be passed to the tool as a comma-separated list:
 //   ./gradlew runVGen '-Pclargs=--arg1=val1,--arg2=val2,--arg3=val3'
+task runCodeGen(type: JavaExec) {
+  classpath = sourceSets.main.runtimeClasspath
+  main = 'com.google.api.codegen.CodeGeneratorTool'
+  if (project.hasProperty('clargs')) {
+    args = clargs.split(',').toList()
+  }
+}
+
+// old task being retained for backwards-friendliness
```

Comment:
Agreed. We should be proactive about our tech debt.

@ethanbao FYI. Once your `toolkit` dependency is updated, we can strip out this code.